docs/zkapps/tutorials/interacting-with-zkapps-server-side.mdx

```diff
@@ -10,279 +16,268 @@ zkApp programmability is not yet available on the Mina Mainnet, but zkApps can n
 
 :::
 
-:::note
+# Interacting with zkApps server-side
 
-This tutorial was last tested with [SnarkyJS](https://www.npmjs.com/package/snarkyjs) 0.8.0.
+While user-facing zkApps can be written for the browser, sometimes it is useful to interact with a zkApp server-side or on your own machine. For example, when initializing a zkApp using programmatically generated information, deploying a zkApp in custom ways, or writing scripts that create transactions depending on real-world or on-chain events.
```

Comment:
I feel like we need a more "why do I care" benefit to this tutorial, what is a practical use case?

Reply:
I guess the simplest real-world use case I could think of would be updating an on-chain value with signed data periodically. Basically a keeper for an oracle. (This would fall under the "creating transactions from real-world events.")